src/runtime/contrib/tflite/tflite_runtime.cc

```diff
@@ -92,20 +91,23 @@ DataType TfLiteDType2TVMDType(TfLiteType dtype) {
   }
 }
 
-
 void TFLiteRuntime::Init(const std::string& tflite_model_bytes,
                          TVMContext ctx) {
   const char* buffer = tflite_model_bytes.c_str();
   size_t buffer_size = tflite_model_bytes.size();
   std::unique_ptr<tflite::FlatBufferModel> model =
     tflite::FlatBufferModel::BuildFromBuffer(buffer, buffer_size);
   tflite::ops::builtin::BuiltinOpResolver resolver;
-  tflite::InterpreterBuilder(*model, resolver)(&interpreter_);
-  ctx_ = ctx;
-}
+  // Build interpreter
+  if (tflite::InterpreterBuilder(*model, resolver)(&interpreter_) != kTfLiteOk) {
```

Comment:
we can define macro for TFLite error checking: `CHECK_STATUS(cond, msg)`

Reply:
thanks for the suggestion, this should be fixed by now